main-app/client/src/components/Navbar/navbar.js

```diff
@@ -1,49 +1,70 @@
 import React from "react";
-import { Link } from "react-router-dom";
-import PropTypes from 'prop-types';
+import { Link, NavLink } from "react-router-dom";
+import "./navbar.scss";
+import PropTypes from "prop-types";
 
-import { UserAuth } from '../../utilities/auth';
-
-import "./navbar.css";
+import { UserAuth } from "../../utilities/auth";
 
 export class NavBar extends React.Component {
   static propTypes = {
-    onLogout: PropTypes.func.isRequired,
-};
+    onLogout: PropTypes.func.isRequired
+  };
 
- logOut() {
+  logOut() {
     this.props.onLogout();
   }
 
   render() {
     // Render either log-in or log-out on nav-bar according to the state
     let logInOutOption;
+    let authNav;
 
-    if(UserAuth.loggedIn() === true) {
+    if (UserAuth.loggedIn() === true) {
+      // if user is logged in, provide option to log out
       logInOutOption = (
-        <div>
-          <Link to="/login" onClick={() => this.props.onLogout()}>
-            Log out
-          </Link>
-        </div>
+        <Link to="/login" onClick={() => this.props.onLogout()}>
+          Log out
+        </Link>
       );
-    }
-    else {
-      logInOutOption = (
-        <div>
-          <Link to="/login">Login</Link>
+      // authorized navigation links, when user is logged in
+      authNav = (
+        <ul className="nav-links-list">
+          <li>
+            <NavLink
+              to="/participants/1"
+              className="btn nav"
+              activeClassName="active"
+            >
+              Home
+            </NavLink>
+          </li>
+          <li>
+            <NavLink to="/intake" className="btn nav" activeClassName="active">
+              Intake Upload
+            </NavLink>
+          </li>
+          <li>
+            <NavLink to="/form" className="btn nav" activeClassName="active">
+              Reporting
```

Comment:
One question though, why is 'Reporting' in the nav pointed to Intake Form?

Can you fix this before merging? Ideally rename `Reporting` to `Intake Form` as we don't have the spec for a Report yet.

Reply:
Hi Tien,

I suppose I was trying to follow along and mirror the design spec as close as possible even though we didn't have some of the page views fleshed out yet. I see now this is probably confusing to anyone aside from myself. I will go ahead and fix that and open a new PR

Cheers